app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -92,7 +93,16 @@ protected void onCreate(final Bundle savedInstanceState) {
 
         sortOnName();
 
-        this.serviceLocator.getBluetoothHandler().start();
+        try {
+            this.serviceLocator.getBluetoothHandler().start();
+        } catch (NoBluetoothException e) {
+            LOGGER.info("Bluetooth not available on device, hiding button");
```

Comment:
I don't think that is necessary, as they are probably aware they don't have Bluetooth, right?